### ISSUE TITLE
Misleading message in logs: error getting the cluster info configmap

### ISSUE THREAD

**Issue by carolynvs** [kubernetes-incubator/service-catalog#1973]
In the service catalog logs, we are logging an error when we can't read the configmap to retrieve the cluster info. I think that is a misleading log message because it's just a warning, not an error, correct? Or do we now require different rbac permissions?

```
controller.go:293] error getting the cluster info configmap: "configmaps \"cluster-info\" is forbidden: User \"system:serviceaccount:svc-cat:service-catalog-controller-manager\" cannot get configmaps in the namespace \"default\""
```

Originally reported in https://github.com/osbkit/minibroker/issues/1

**Comment by MHBauer** [comment]
must have forgot that in #1658

**Comment by MHBauer** [comment]
closed by #1974, hopefully

**Comment by jboyd01** [comment]
still seeing this issue
